### ISSUE TITLE
Questions about the cross entropy loss

### ISSUE THREAD

**Issue by HuayuLiArizona** [DefangChen/SimKD#9]
Greetings,

I hope this message finds you well. 

I was wondering if you could kindly clarify something for me. 

Based on the command "python train_student.py --path_t ./save/teachers/models/resnet32x4_vanilla/resnet32x4_best.pth --distill simkd --model_s resnet8x4 -c 0 -d 0 -b 1 --trial 0", it seems that the student is solely trained with L2 loss between teacher and student features, without the classification loss of student output logits. 

However, I wanted to confirm if my understanding is correct.

Thank you for your time and assistance.

**Comment by DefangChen** [comment]
Yes, you are right.
